src/GitExtensions.PluginManager/Plugin.cs

```diff
@@ -64,7 +61,10 @@ public override bool Execute(GitUIEventArgs gitUiCommands)
                 Arguments = args.ToString(),
                 UseShellExecute = false,
             };
-            Process.Start(info);
+
+            var process = Process.Start(info);

```

Comment:
In this case we'd want to dispose the process
```suggestion
            using var process = Process.Start(info);

```